Autopilot/stm32f7-openocd.cfg

```diff
@@ -0,0 +1,12 @@
+# This is an STM32F7 discovery board with a single STM32F756NGH6 chip.
+# http://www.st.com/web/catalog/tools/FM116/SC959/SS1532/LN1848/PF261641
+
+# This is for using the onboard STLINK/V2
+source [find interface/stlink-v2.cfg]
+
+transport select hla_swd
+
+# increase working area to 256KB
+set WORKAREASIZE 0x40000
```

Comment:
What does `WORKAREASIZE` represent? Is it the flash space of the chip? Might need to be different for our board.

Reply:
hmm.. From [here](http://openocd.org/doc/html/Config-File-Guidelines.html):

Reply:
This cfg file (and so the 0x40000) was copied from a config file for the stm32f7discovery, which uses the same CPU I believe.